The framework build of Python on the Mac provides a pythonw for doing
graphical applications.

--- python.eselect
+++ python.eselect
@@ -46,7 +46,7 @@
 # Try to remove python and python.1 symlinks
 remove_symlinks() {
 	local symlink symlink_target symlink_target_found
-	rm -f "${INTERPRETER_PATH}"{idle,pydoc,python,python-config} &>/dev/null && \
+	rm -f "${INTERPRETER_PATH}"{idle,pydoc,python,pythonw,python-config} &>/dev/null && \
 	rm -f "${MAN_PATH}"python.1{,.gz,.bz2,.lzma,.xz} &>/dev/null
 	for symlink in "${INTERPRETER_PATH}python"?; do
 		[[ ! -L "${symlink}" ]] && continue
@@ -105,6 +105,10 @@
 		if [[ -f "${target/python/2to3-}" ]]; then
 			ln -nfs "${target/python/2to3-}" 2to3
 		fi
+		# for Mac framework builds
+		if [[ -f "${target/python/pythonw}" ]] ; then
+			ln -nfs "${target/python/pythonw}" pythonw
+		fi
 
 		popd 1>/dev/null
 	else
